Error running main function as admin
Traceback (most recent call last):
  File "C:\Users\shelly ben zion\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\pyuac\main_decorator.py", line 89, in main_requires_admin
    rv = run_function(*args, **kwargs)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\shelly ben zion\Desktop\Cyber project\UpToDater\Code\client.py", line 111, in open_dll_exe
    os.system(['start', '/min', "", 'OpenHardwareMonitor.exe'])
TypeError: system() argument 'command' must be str, not list
Error running main function as admin
Traceback (most recent call last):
  File "C:\Users\shelly ben zion\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\pyuac\main_decorator.py", line 89, in main_requires_admin
    rv = run_function(*args, **kwargs)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\shelly ben zion\Desktop\Cyber project\UpToDater\Code\client.py", line 113, in open_dll_exe
    subprocess.Popen([exe_path])
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1008.0_x64__qbz5n2kfra8p0\Lib\subprocess.py", line 1024, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1008.0_x64__qbz5n2kfra8p0\Lib\subprocess.py", line 1509, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
PermissionError: [WinError 5] Access is denied
